backends/occa/include/ceed.h

```diff
@@ -0,0 +1,4 @@
+#ifndef _OCCA_INCLUDE_CEED_H_
```

Comment:
If you want to include this during JiT, see the example files like this in `/include/ceed/jit-source/cuda/cuda-jit.h'

Any files that will be used as JiT source should go in `include/ceed/jit-source/occa`